CommunityProjects.md

```diff
@@ -8,30 +8,40 @@ SPDX-FileCopyrightText: 2020 seL4 Project a Series of LF Projects, LLC.
 This page collects seL4-based projects. If you know of a project that
 you'd like to be listed here, add it.
 
+## Application areas
 The seL4 kernel is currently used in projects in the following application areas:
 
 - Automotive
-- Aviation (e.g.,
-        [SMACCM](https://trustworthy.systems/projects/TS/SMACCM/))
+- Aviation
+  - [SMACCM](https://trustworthy.systems/projects/TS/SMACCM)
 - Connected Consumer Devices
 - SCADA
-- Spaceflight (e.g., the
-        [UNSW QB50](https://trustworthy.systems/projects/TS/qb50) satellite.)
+  - [Laot](https://trustworthy.systems/projects/TS/laot)
+- Spaceflight
+  - [UNSW QB50](https://trustworthy.systems/projects/TS/qb50) satellite
+- Virtualization
+  - [in general](https://trustworthy.systems/projects/TS/virtualisation/about)
+      and especially for [Linux](https://trustworthy.systems/projects/TS/virtualisation)
 
-The following platforms and language runtimes have been or are in the
-process of being ported to seL4:
+See also the list of [Projects at Trustworthy Systems](https://trustworthy.systems/projects)
 
-- [CamkES](https://trustworthy.systems/projects/TS/trustcomp)
-- [Genode](http://genode.org/about/index)
-- [Linux (virtualised)](https://trustworthy.systems/projects/TS/virtualisation/about)
-- [WasmEdge](https://github.com/WasmEdge/WasmEdge): [porting a Cloud-native WebAssembly Runtime to seL4](https://github.com/second-state/wasmedge-seL4)
-- C
-- [Ivory](http://ivorylang.org/ivory-introduction.html)
+## OS Personalities and Frameworks
+- [CamkES](https://trustworthy.systems/projects/TS/camkes) to build
+    [Trustworthy components](https://trustworthy.systems/projects/TS/trustcomp)
+- [Genode](https://genode.org)
+- [Neptune OS](https://github.com/cl91/NeptuneOS) is a WinNT personality
+- [seL4 Core Platform](https://github.com/BreakawayConsulting/sel4cp)
+
+## Languages and Runtimes
+
+- C is the default seL4 language
+- C++
 - Rust:
-  - The [Robigalia](https://rbg.systems) project:
+  - The [Robigalia](https://gitlab.com/robigalia) project:
```

Comment:
There seems nothing wrong with <https://rbg.systems>, why the update?

Reply:
> There seems nothing wrong with https://rbg.systems, why the update?

The link was dead when I checked. Seems it's working again now.

Reply:
Got it. It's relatively new (updated from the old robigalia domain), so I think it's supposed to be working. Might have been a glitch.

Reply:
reverted